DOCS.md

```diff
@@ -5,7 +5,9 @@ You can override the default configuration with the following parameters:
 
```

Comment:
You can drop this file and create a pr to the plugin index

Reply:
I'll create a PR for the plugin index and remove this and fix the References in the README once it's merged :)